README.md

```diff
@@ -34,12 +34,43 @@ uses: actions/checkout@v2
 with:
   fetch-depth: 0
 ```
-# Optional Parameters
+3. To publish security issues to ASoC when a pull request is merged, add the following content to your workflow file:
+```yaml
+on:
+  pull_request:
+    types: [closed]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Publish issues to ASoC
+        uses: HCL-TECH-SOFTWARE/appscan-codesweep-action@v2
+        with:
+          asoc_key: ${{secrets.ASOC_KEY}}
+          asoc_secret: ${{secrets.ASOC_SECRET}}
+          publish_on_merge: true
+          application_id: <your ASoC application ID>
+    env: 
+      GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+```
+# Optional Parameters For Scanning
 - status - The status of the checks if any security issues are found. Must be one of 'action_required', 'failure', or 'neutral'. The default is neutral. For example:
 ```yaml
 with:
   status: failure
 ```
+# Optional Parameters For Publishing Issues to AppScan on Cloud
+- issue_status - The status of issues that are published to ASoC. Must be one of 'new', 'open', 'inprogress', 'noise', 'fixed', or 'passed'. The default is 'open'.
```

Comment:
I think ASoC design wants to move away from using "New". I will check with Julie and confirm if we can keep this status.

Reply:
We should remove the 'new' status. Keep the rest.